scripts/install_server_from_src.sh

```diff
@@ -15,67 +14,133 @@ set -e
 : ${DOWNLOAD_CLIENT_TEMPLATES:=true}
 : ${PYTHON_INSTALL:=true}
 : ${DOCKER:=false}
+: ${INSTALL_BIN:="install"}
+: ${INSTALL_OPTS:="-p -m644"}
+: ${SRC_DIR:="/grr"}
+
+function header()
+{
+  echo ""
+  echo "##########################################################################################"
+  echo "     ${*}";
+  echo "##########################################################################################"
+}
+
+OPTIND=1
+while getopts "h?ls:" opt; do
```

Comment:
Yeah, OK. Looks like you need to do silly stuff to pass env var's to vagrant provision script. If you have time to convert the other variables and call sites to flag options feel free to, but I'm also fine with just adding a TODO above the variables that need to be replaced.